### ISSUE TITLE
The Destroy method removes inline styling

### ISSUE THREAD

**Issue by Superhands89** [NickPiscitelli/Glider.js#119]
I have a glider set up whereby each slide has a background image.
When the glider is destroyed, the background images are removed.
Please see my demo of this on codepen...

https://codepen.io/SuperHands/pen/bGpeQgQ

My expectation is that when the glider is destroyed, the elements will return to their original state before the glider was initialized (including inline styling such as background images.)

Best,

**Comment by NickPiscitelli** [comment]
I disagree that this should be fixed. It is expected that the plugin will
manipulate the state of the element it is initialized on.

To work around this, simply leave the styling of the initialized container
bare and apply any custom rules applied with wrapping containers.

If there is no control of the rendered HTML, then save the element state in
JS before initializing and restore after a destroy.

**Comment by NickPiscitelli** [comment]
I misunderstood the original issue. I will look into this, this should be possible to fix.

**Comment by NickPiscitelli** [comment]
This might be difficult to fix without bloating the plugin. As of now, the style attribute is removed to unset any custom rules applied to the element via Glider.

An easy workaround should be to just wrap the background images with a containing div.

**Comment by NickPiscitelli** [comment]
At the moment, it's a bit overzealous and strips all inline styles of all descendants, but it should be an easy fix to just apply to only the track and items.

**Comment by Superhands89** [comment]
Thanks for your speedy response!

Is this something you are looking into or you thinking this is still an enhancement?

**Comment by NickPiscitelli** [comment]
Sorry, I thought I had replied to this e-mail, but I see it's still left in drafts.

I'll look into adding this in the next version release, but for now, I'd recommend just overriding the destroy method to the following:

```
gliderPrototype.destroy = function () {
    var _ = this

    var replace = _.ele.cloneNode(true)

    var clear = function (ele) {
      ele.removeAttribute('style');
      [].forEach.call(ele.classList, function (className) {
        /^glider/.test(className) && ele.classList.remove(className)
      })
    }
    // remove track
    replace.children[0].outerHTML = replace.children[0].innerHTML
    clear(replace);
    [].forEach.call(replace.children, clear)
    _.ele.parentNode.replaceChild(replace, _.ele)
    _.event(_window, 'remove', {
      resize: _.resize
    })
    _.emit('destroy')
  }
```

This should only clear the styling of the direct descendants as opposed to all of the children within the slides.
